modules/transport-netty4/src/main/java/org/opensearch/http/netty4/Netty4HttpResponse.java

```diff
@@ -46,7 +46,7 @@ public class Netty4HttpResponse extends DefaultFullHttpResponse implements HttpR
 
     private final HttpHeaders requestHeaders;
 
-    Netty4HttpResponse(HttpHeaders requestHeaders, HttpVersion version, RestStatus status, BytesReference content) {
+    public Netty4HttpResponse(HttpHeaders requestHeaders, HttpVersion version, RestStatus status, BytesReference content) {
```

Comment:
The `Netty4HttpResponse` should be package protected - this is 100% internal  plugin implementation

Reply:
Removed the change to the access modifier.